tools/tidb-binlog-cluster.md

```diff
@@ -305,7 +305,7 @@ The following part shows how to use Pump and Drainer based on the nodes above.
 
     - Taking deploying Pump on "192.168.0.11" as an example, the description for command line parameters of Pump is as follows:
 
-        ```
+        ```bash
```

Comment:
Well this would be superfluous. The content is plain-text not a bash script, highlighting it as bash would make the content harder to read.

```suggestion
        ```
```